src/test/test_portfolio.py

```diff
@@ -158,6 +158,38 @@ def test_sell_limit_order(self):
         finally:
             pass
 
+    def test_get_total_fees_paid(self):
+        date = datetime.datetime(2021, 6, 14, 15, 57)
+        data_source = FmpDataSource(os.getenv("FMP_TOKEN"), Resolution.MINUTE)
```

Comment:
Can you mock the price value? We don't need to test the API calls to FMP here. Otherwise, everything else is very good.

Reply:
Good point, thanks!